src/app/ui/bridge/BridgePage.style.ts

```diff
@@ -2,12 +2,14 @@
 
 import type { BoxProps } from '@mui/system';
 import { Box, styled } from '@mui/system';
-import type { Breakpoint } from '@mui/material/styles';
 
 export const BridgePageContainer = styled(Box, {
   shouldForwardProp: (prop) => prop !== 'active',
```

Comment:
do we still need this one? Same comment as Seb, so might be dumb

Reply:
No, this should also be removed. Thanks! It´s cleaned